### PR TITLE
feat: webp wishlist

### PR DESCRIPTION
**Pull request by csandru-plenty**
### All changes meet the following requirements
- [ ] Changelog entry was added
- [ ] Changes have been documented
- [ ] Changes have been tested by the author
- [ ] Changes have been tested by the reviewer
- [ ] Changes to SCSS have been accounted for in plentyShop LTS Modern

@plentymarkets/plentyshop

### CODE REVIEW

File: package-lock.json
Comment (ksted) [inline package-lock.json:1]: **chore**: Revert

I assume updating the `package-lock.json` was an accident. When working in Ceres, you have to use a lower Node version, like 16 or 14.

**Review by ksted — CHANGES_REQUESTED** [review]
**chore**: Please revert the formatting changes, so that the changed files follow the same style as the rest of the code base.

File: package-lock.json
Comment (csandru-plenty) [reply package-lock.json:1]: yeah, reverted

File: resources/lang/de/Config.properties [hunk 1]
```diff
@@ -106,6 +106,7 @@ globalMaxAge1Week = 1 Woche
 globalMaxAgeInfinite = Immer
 globalRegistrationRequirePrivacyPolicyConfirmation = Zustimmung zur Datenschutzerklärung bei der Kundenregistrierung voraussetzen
 globalBlockCookies = Nicht akzeptierte Cookies blockieren (empfohlen)
+globalWebpImagesEnabledLabel = Aktivieren Sie die Verwendung von WebP-Bildern
```
Comment (ksted) [inline resources/lang/de/Config.properties]:
```suggestion
globalWebpImagesEnabledLabel = WebP-Format für Artikelbilder verwenden
```

File: resources/lang/en/Config.properties [hunk 2]
```diff
@@ -104,6 +104,7 @@ globalMaxAge1Week = 1 week
 globalMaxAgeInfinite = Always
 globalRegistrationRequirePrivacyPolicyConfirmation = Require acceptance of privacy policy for customer registration
 globalBlockCookies = Block unaccepted cookies (recommended)
+globalWebpImagesEnabledLabel = Enable webp images usage
```
Comment (ksted) [inline resources/lang/en/Config.properties]:
```suggestion
globalWebpImagesEnabledLabel = Use WebP format for item images
```

File: src/Wizard/ShopWizard/Steps/Builder/OnlineStoreStep.php [hunk 3]
```diff
@@ -381,6 +382,24 @@ private function buildSessionLifeTimeSection(): array
         ];
     }
 
+    private function buildWebpImageSettings(): array
+    {
+        return [
+            "title" => "Config.globalWebpImagesEnabledLabel",
+            "description" => "Config.globalWebpImagesEnabledLabel",
```
Comment (ksted) [inline src/Wizard/ShopWizard/Steps/Builder/OnlineStoreStep.php:389]: **todo**: The description should provide some additional information about the feature and give customers a reason to enable the option, as well as inform them about scenarios when they shouldn't activate it.

We can do this in a follow-up PR.

**Review by ksted — DISMISSED** [review]

**Review by ksted — APPROVED** [review]

**Pull request merged**